fio-plot/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:3.7.8-slim
+
+ENV DEBIAN_FRONTEND=noninteractive
+ARG VERSION=3.12-2
+
+ADD . /
+
+WORKDIR /
+
+RUN apt update && \
+    apt install --no-install-recommends -y git fio=${VERSION} && \
+    git clone https://github.com/louwrentius/fio-plot && \
+    rm -rf fio-plot/images && \
+    pip install -r fio-plot/fio_plot/requirements.txt && \
```

Comment:
this can include `--no-cache-dir` to avoid creating a cache directory with package dependencies. And `-r` can be given multiple times, so it can be one line

Reply:
thanks